File: packages/insomnia-httpsnippet/test/fixtures/output/objc/nsurlsession/query.m
#import <Foundation/Foundation.h>

NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:[NSURL URLWithString:@"http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value"]
                                                       cachePolicy:NSURLRequestUseProtocolCachePolicy
                                                   timeoutInterval:10.0];
[request setHTTPMethod:@"GET"];

NSURLSession *session = [NSURLSession sharedSession];
NSURLSessionDataTask *dataTask = [session dataTaskWithRequest:request
                                            completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
                                                if (error) {
                                                    NSLog(@"%@", error);
                                                } else {
                                                    NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *) response;
                                                    NSLog(@"%@", httpResponse);
                                                }
                                            }];
[dataTask resume];

File: packages/insomnia-httpsnippet/test/fixtures/output/node/request/query.js
var request = require("request");

var options = { method: 'GET',
  url: 'http://mockbin.com/har',
  qs: { foo: [ 'bar', 'baz' ], baz: 'abc', key: 'value' } };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

